### ISSUE TITLE
Design | MHV&DS Logon Retirement | February-March initial messaging (awareness + encouragement)

### ISSUE THREAD

**Issue by mtcA6** [department-of-veterans-affairs/va.gov-team#73358]
## Background 
DS logon and MHV credentials are being retired and we need to display alerts advising users to update to the alternative login options. 

We can complete this work in advance of the press release, everything hinges on the press release advising the public that these services will be retired. 

The messaging should be short and should link to other longform pages (eg. FAQ page) 

1. Figure out where we're going to put messaging
     - Message at the top of the hub
     - Message in the Account Security Page 
     - Where else? 

2. Figure out what we're linking to? an FAQ page 

3. Determine what to display for LOA1 vs LOA3, is it the same? 
     - for loa1 do we say hey don't verify with these credentials (may already say not to use these but double check this) 

4. How will we change this as the severity/urgency of the migration increases? [nice to have/be thinking about] 
     - use the same language that has already been approved  

### Account Security Maintenance 
Since we know we're going to adjust the account security page for the credential retirement, we should incorporate the maintenance ticket that is currently in the backlog. I'm (mtcA6) closing the ticket and incorporating the tasks below but you can see the [original ticket here](https://github.com/department-of-veterans-affairs/va.gov-team/issues/74053). 

#### Background from #74053
While reviewing the process list headings ([ticket](https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2383)), Danielle noticed that we're still using the phrase "2-factor authentication" on the account security page said it should be multifactor authentication instead.  Let's work with her to get our page reflecting the latest content guidelines.

We can also use this ticket to cover the work if the design system a11y folks determine we shouldn't have headings as links on this page.

## other stakeholders 
`#secure-sign-in-transition`

## Tasks 
- [ ] Whoever picks this up can remove the other designer from the assignees
- [ ] Work with Danielle Thierry to refine the content for the current 2FA step, when it is both complete and incomplete.
- [ ] If needed, work with Danielle to refine all the content so headings are no longer links, and the steps themselves contain the calls to action
- [ ] Update Figma mock-ups
- [ ] Move to validate so Travis can create the FE ticket for the work

## Acceptance Criteria 
- [x] [designs for FE to work from ](https://www.figma.com/file/OmY3qqcBnK043nnnUvlsjh/%2373358---Auth-Exp-Credential-Retirement-Phase-1?type=design&node-id=1-2509&mode=design&t=YpRnnYjiw7Ufc97L-0)

## Output 
[Figma file](https://www.figma.com/file/OmY3qqcBnK043nnnUvlsjh/%2373358---Auth-Exp-Credential-Retirement-Phase-1?type=design&node-id=1-2509&mode=design&t=YpRnnYjiw7Ufc97L-0)

**Comment by mtcA6** [comment]
image shared by Lauren from the `#secure-sign-in-transition` team, everything is dependent upon the press release. 

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/129431463/642b3512-cf3c-4c18-a0ea-9415129a39df)

**Comment by andaleliz** [comment]
@Samara-Strauss @mtcA6 as I think about this, I want to check an assumption as I think through my recommendations of where we should put this. I'm assuming this is phase 1 of the awareness messaging and VA.gov will get more aggressive with how "in your face" this is as the deadline approaches. Is that right?

**Comment by mtcA6** [comment]
You're assumption re: increased aggressiveness is spot on, it's on the epic collapsed under draft plans (I don't know if that draft was finalized by octo which is why I left it as "draft", @Samara-Strauss did they finalize that draft project plan?) 

> When:
> o February-March 2024: Add initial messaging (awareness + encouragement)
> o August 2024: Increase urgency of messaging (encouragement + get help now)
> o November 2024: Increase urgency of messaging (alert + get help now)
> o After December 31, 2024: Adjust messaging to post-transition messaging.

As for the latter comment, I agree w/your judgement call there lets keep BAI prioritized over the DS Logon Retirement Item 

Related but unrelated, I'm trying to pull together the staging users that can be used to view our error cases/states for our contact center guide (and other reasons as well making QA testing easier, checking error states for USWDS changes etc). Do you happen to know if there's a staging user that works for viewing the BAI?

**Comment by Samara-Strauss** [comment]
Yes, the messaging should get more aggressive over time. I would say so should the treatment. We don't want a notification sitting in people's accounts for a year that eventually becomes noise.

IMO, I think initial messaging should be in-context of where we prompt people to change their sign-in information. Over time, we could add a louder notification to the top of the page(s).

Of course, final call on treatment is up to you, but I feel like if I have a vision, I should say it instead of making you try to guess what's in my head :) 

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/34068740/9c935024-9233-47e2-b048-29d7633cb37d)

![image](https://github.com/department-of-veterans-affairs/va.gov-team/assets/34068740/93274d97-45e5-4d44-884c-9a38963870fd)

**Comment by andaleliz** [comment]
Thanks to both of you for weighing in.  Here's what I've drafted up for the content review tomorrow, for folks using the MHV or DS Logon credential.  Since these are for content purposes at this point, the mocks have varying levels of completeness for surrounding components. I'll get those cleaned up for FE once the content is approved.

1. LOA1 - account security: display a full alert component on the account security page, [as shown here](https://www.figma.com/file/OmY3qqcBnK043nnnUvlsjh/%2373358---Auth-Exp-Credential-Retirement-Phase-1?type=design&node-id=1-2509&mode=design&t=SoBtvAtW6gBrGAxV-11). LOA1 people have access to no other part of profile. We drafted this up a while back but never implemented it. I believe the content is already approved but I'll double check. 
2. LOA3 - Account security: display a dismissible slim alert in the sign-in information box on account security ([example](https://www.figma.com/file/OmY3qqcBnK043nnnUvlsjh/%2373358---Auth-Exp-Credential-Retirement-Phase-1?type=design&node-id=1-7385&mode=design&t=SoBtvAtW6gBrGAxV-11)). I do think an alert component is warranted because otherwise it's going to be really hard to notice on the page.
3. LOA3 - Contact information: same treatment as account security, but in the sign-in email section. [Mock-up](https://www.figma.com/file/OmY3qqcBnK043nnnUvlsjh/%2373358---Auth-Exp-Credential-Retirement-Phase-1?type=design&node-id=1-1587&mode=design&t=SoBtvAtW6gBrGAxV-11)
4. LOA3 - Profile hub: Reuse the slim dismissible alert component at the top of the page, [like so.](https://www.figma.com/file/OmY3qqcBnK043nnnUvlsjh/%2373358---Auth-Exp-Credential-Retirement-Phase-1?type=design&node-id=1-992&mode=design&t=SoBtvAtW6gBrGAxV-11) IMO, we should add this here because people could easily miss the information in the other two sections of the profile, and it feels important especially given the struggles I've heard MHV users have with the other credentials.  However, if you disagree and feel this is better for a Phase 2, I can be convinced that's the right way to go.

If you have a chance to provide feedback before 10am tomorrow, that would be wonderful! Otherwise, I'll take these to content review and incorporate any changes along with the content edits.

**Comment by Samara-Strauss** [comment]
I am so sorry I am getting you feedback after content office hours 🤦‍♀️ Thank you for being prepared and going to this meeting. How did it go?

I am OK with this general approach. Two thoughts:

1. I don't think we need anything on the profile hub for now.
  - Side question: Do we show BAI on the profile hub?
2. Idk what Danielle did/did not improve in terms of copy, but I think we could shorten this a bit by removing the sentence "If you don't have one of these accounts..." (keep the CTA links).

**Comment by andaleliz** [comment]
@Samara-Strauss now I'm the one who is sorry for letting this sit all day - I replied to you only in my mind. The meeting went great! Danielle provided a few small changes but agrees with the approach.

>I don't think we need anything on the profile hub for now.

- Noted, and removed from mock-ups

>Side question: Do we show BAI on the profile hub?
- Yes

>Idk what Danielle did/did not improve in terms of copy, but I think we could shorten this a bit by removing the sentence "If you don't have one of these accounts..." (keep the CTA links).

She did make some updates to the copy, which is reflected here. We updated the CTA link to point to a resources and support page, so it is shorter. I think we need the "if you don't have..." copy for that link to make sense, at least in the LOA3 alerts ([example](https://www.figma.com/file/OmY3qqcBnK043nnnUvlsjh/%2373358---Auth-Exp-Credential-Retirement-Phase-1?type=design&node-id=0-1&mode=design&t=BCxcHTZqb167wB9F-11)).  What do you think?

I'm still waiting for her to respond to a change on the LOA1 alert before moving this to validate so FE tickets can be made.

**Comment by andaleliz** [comment]
I followed up with Danielle again and haven't heard back. The change I'm waiting for her to respond on is very small, and moves us closer to being consistent with other areas so I'm 99% sure she will not have an issue with it. @mtcA6 I feel good about sending this to FE, if you do too. Worse case scenario we have to reverse the order of words in one sentence, in one alert, later. Lmk what you thinkl

**Comment by Samara-Strauss** [comment]
IMO we can move forward. Thanks, Liz!

**Comment by andaleliz** [comment]
@Samara-Strauss wonderful! 

All of the updates are in [this Figma file](https://www.figma.com/file/OmY3qqcBnK043nnnUvlsjh/%2373358---Auth-Exp-Credential-Retirement-Phase-1?type=design&node-id=1%3A2509&mode=design&t=72D4ptOfW79bdUnG-1), as well as in the respective files for the product. Once this is on the FE, I will delete the ticket-specific Figma file.  

Moving to validate.

**Comment by mtcA6** [comment]
@andaleliz can you incorporate the additional maint. feedback that we have in the story? 



> Account Security Maintenance
> Since we know we're going to adjust the account security page for the credential retirement, we should incorporate the maintenance ticket that is currently in the backlog. I'm (mtcA6) closing the ticket and incorporating the tasks below but you can see the https://github.com/department-of-veterans-affairs/va.gov-team/issues/74053.
> 
> Background from https://github.com/department-of-veterans-affairs/va.gov-team/issues/74053
> While reviewing the process list headings (https://github.com/department-of-veterans-affairs/vets-design-system-documentation/issues/2383), Danielle noticed that we're still using the phrase "2-factor authentication" on the account security page said it should be multifactor authentication instead. Let's work with her to get our page reflecting the latest content guidelines.
> 
> We can also use this ticket to cover the work if the design system a11y folks determine we shouldn't have headings as links on this page.

**Comment by andaleliz** [comment]
@mtcA6 thanks for the reminder! I did update all the designs to replace "2-factor" with multi-factor, so that's good to go. I did not hear back from anyone about the headings as links. I suspect that will be a larger piece of work if they do want us to change it and I would recommend NOT following my original guidance of using this ticket to cover that same work in favor of keeping things separate and more easily trackable.
